src/components/v5/payments/TmpAdvancedPayments.tsx

```diff
@@ -533,45 +577,89 @@ const TmpAdvancedPayments = () => {
-        >
-          Create streaming payment
-        </ActionButton>
+        <div className="flex flex-col gap-4">
+          <div className="flex gap-4">
+            <InputBase
+              onChange={(e) => setTokenAddress(e.currentTarget.value)}
+              value={tokenAddress}
+              label="Token Address"
+            />
+            <InputBase
+              onChange={(e) => setDecimalAmount(e.currentTarget.value)}
+              value={decimalAmount}
+              label="Token Decimals"
+            />
+            <InputBase
+              onChange={(e) => setTransactionAmount(e.currentTarget.value)}
+              value={transactionAmount}
+              label="Transaction Amount"
+            />
+            <InputBase
+              onChange={(e) => setAnnotation(e.currentTarget.value)}
+              value={annotation}
+              label="Annotation"
+              placeholder="Annotation"
+              type="text"
+            />
+          </div>
+          <div className="flex gap-2">
+            <ActionButton
+              actionType={ActionTypes.EXPENDITURE_CREATE}
+              values={createExpenditurePayload}
+            >
+              Create expenditure
+            </ActionButton>
+            <Button onClick={handleCreateStakedExpenditure}>
+              Create staked expenditure
+            </Button>
+            <ActionButton
+              actionType={ActionTypes.EXPENDITURE_CREATE}
+              values={createStagedExpenditurePayload}
+            >
+              Create staged expenditure
+            </ActionButton>
+            <ActionButton
+              actionType={ActionTypes.STREAMING_PAYMENT_CREATE}
+              values={createStreamingPaymentPayload}
+            >
+              Create streaming payment
+            </ActionButton>
+            {latestTxHash && (
+              <button
+                type="button"
+                className="relative flex flex-col gap-4"
+                onClick={() => {
+                  navigator.clipboard.writeText(latestTxHash.toString());
+                  setCopiedMsgVisible(true);
+                  setTimeout(() => {
+                    setCopiedMsgVisible(false);
+                  }, 1000);
+                }}
+              >
+                <p className="text-sm font-bold">
+                  Latest Transaction Hash (click to copy)
+                </p>
+                <p className="flex-1 cursor-pointer text-sm">{latestTxHash}</p>
+                <div
+                  className={clsx(
+                    'bg-base-white/20 pointer-events-none absolute flex h-full w-full items-center justify-center opacity-0 backdrop-blur-sm transition-opacity',
+                    {
+                      'opacity-100': copiedMsgVisible,
+                    },
+                  )}
+                >
+                  <p className="font-bold">Copied to clipboard! 😎</p>
+                </div>
+              </button>
+            )}
```

Comment:
I liked the ease of having clickable links to a motion logged in the console, do you think this could redirect to the action with latest tx hash instead?

Reply:
On it!